idaes/power_generation/carbon_capture/mea_solvent_system/property_package/liquid_prop.py

```diff
@@ -0,0 +1,1282 @@
+                             doc='thermal conductivity of fluid [W/m.K]')
+
+    def _surf_tens(self):
+        '''
+        Surface tension,N/m
+        '''
+        def rule_surf_tens_comp(blk,j):
+                r  = blk.mass_frac1['MEA']
+                T  = blk.temperature
+                Tc = blk._params.temperature_crit
+                if j in blk._params.component_list_solvent:
+                    return blk._params.surf_tens_param[j,1]*(1-T/Tc[j])**\
+                          (blk._params.surf_tens_param[j,2] +
+                           blk._params.surf_tens_param[j,3]*T/Tc[j] +
+                           blk._params.surf_tens_param[j,4]*(T/Tc[j])**2)
+                elif j == 'CO2':
+                    return blk._params.surf_tens_param[j,1]*r**2 + \
+                           blk._params.surf_tens_param[j,2]*r + \
+                           blk._params.surf_tens_param[j,3] +\
+                           T*(blk._params.surf_tens_param[j,4]*r**2 +
+                           blk._params.surf_tens_param[j,5]*r +
+                           blk._params.surf_tens_param[j,6])
+
+        self.surf_tens_comp = Expression(self._params.component_list,
+                                         rule = rule_surf_tens_comp,
+                                          doc="surface tension of components"
+                                              "[N/m]" )
+        def rule_surf_tens(blk):
+                r  = blk.mass_frac1['MEA']
+                alpha = self.mole_frac['CO2']/blk.mole_frac['MEA']
+                Fa = blk._params.surf_tens_F_param[1]
+                Fb = blk._params.surf_tens_F_param[2]
+                Fc = blk._params.surf_tens_F_param[3]
+                Fd = blk._params.surf_tens_F_param[4]
+                Fe = blk._params.surf_tens_F_param[5]
+                Ff = blk._params.surf_tens_F_param[6]
+                Fg = blk._params.surf_tens_F_param[7]
+                Fh = blk._params.surf_tens_F_param[8]
+                Fi = blk._params.surf_tens_F_param[9]
+                Fj = blk._params.surf_tens_F_param[10]
+                return blk.surf_tens_comp['H2O'] + \
+                    (blk.surf_tens_comp['CO2'] - blk.surf_tens_comp['H2O'])*\
+                    blk.mole_frac['CO2']*(Fa+Fb*alpha+Fc*alpha**2+Fd*r+Fe*r**2)\
+                    + (blk.surf_tens_comp['MEA'] - blk.surf_tens_comp['H2O'])*\
+                   (Ff + Fg*alpha + Fh*alpha**2 + Fi*r + Fj*r**2)*\
+                   blk.mole_frac['MEA']
+
+        self.surf_tens = Expression(rule = rule_surf_tens,
+                                    doc="surface tension [N/m]")
+
+    def _pressure_sat(self):
+        '''
+        saturated vapor pressure of solvents
+        '''
+        def rule_pressure_sat_comp(blk,j):
+                T = blk.temperature
+                return (exp(blk._params.pressure_sat_param[j,1]
+                          + blk._params.pressure_sat_param[j,2]/T +
```

Comment:
Inconsistent placement of the operators - PEP8 puts operators at the end of the line when doing a line break, not the beginning of the new line.

Reply:
Resolved with PEP8 Linting